src/main/java/uk/gov/hmcts/reform/judicialapi/JudicialApplication.java

```diff
@@ -2,8 +2,16 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.cloud.openfeign.EnableFeignClients;
+import uk.gov.hmcts.reform.authorisation.ServiceAuthorisationApi;
+import uk.gov.hmcts.reform.idam.client.IdamApi;
 
 @SpringBootApplication(scanBasePackages = "uk.gov.hmcts.reform.judicialapi")
+@EnableCaching
```

Comment:
why we need @Enablecaching?

Reply:
Thought we might need it in the future, removed it now.